src/Map/Map.jsx

```diff
@@ -290,13 +329,19 @@ export class Map extends Component {
   }
 
   render () {
-    const {center, mapOptions, tileLayerOptions, wms, zoomControlPosition} = this.props
+    const {
+      center,
```

Comment:
You are right. I had a problem when I was testing it with DefaultChart because it was sending in nulls and not providing a default center point. I will switch that.